docusaurus/static/recipes-assets/splash-screen/SplashScreen.module.css

```diff
@@ -0,0 +1,41 @@
+@import "../../styles/imports/variables";
+
+:root {
+    --z-index-max: 2147483647;
+    --color-background: #221240;
+    --color-text: #fff;
+    --color-progress-bar: #f64f51;
+}
+
+.splashScreen {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    z-index: var(--z-index-max);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--color-background);
+    color: var(--color-text);
+    transition: opacity ease 0.25s 0.2s, visibility 0s 0.45s;
```

Comment:
nitpick: the shortcut order normally used for transitions is 
`transition: transition-property transition-duration transition-timing-function transition-delay;`

like this:
`transition: opacity 0.25s ease 0.2s, visibility 0s ease 0.45s;`

Reply:
Done, can you recheck?